src/Compressors/Gaussian.jl

```diff
@@ -0,0 +1,98 @@
+struct Gaussian <: Compressor
+    n_rows::Int64
+    n_cols::Int64
+end
+
+function Gaussian(;n_rows::Int64 = 0, n_cols::Int64 = 0)
+    # Partially construct the Gaussian datatype
+    return Gaussian(n_rows, n_cols)
+end
+
+mutable struct GaussianRecipe <: CompressorRecipe
+    n_rows::Int64
+    n_cols::Int64
+    scale::Float64
+    sketch_matrix::Matrix{Float64}
+end
```

Comment:
Terminology should be kept the same across compression methods. E.g., `op` in SparseSign, whereas here it is `sketch_matrix`